### PR TITLE
Minor: doc : python 3.2 or newer

### PR DESCRIPTION
**Pull request by sciunto**

### CODE REVIEW

**Comment by brycied00d** [comment]
This PR makes no sense. r2e requires at least Python version 3.2, as demonstrated by the ` if _sys.version_info < (3, 2): raise ImportError`

**Comment by sciunto** [comment]
read again the PR.

File: rss2email/__init__.py [hunk 1]
```diff
@@ -49,5 +49,5 @@
 
 if _sys.version_info < (3, 2):
     raise ImportError(
-        "rss2email requires Python 3.2, but you're using:\n{}".format(
-            _sys.version))
+        "rss2email requires Python 3.2 or older, but you're
```
Comment (brycied00d) [inline rss2email/__init__.py]: rss2email is incompatible with versions of Python older than 3.2. It requires Python 3.2 or *newer*

**Comment by sciunto** [comment]
It was a typo

**Comment by brycied00d** [comment]
Well now it makes a whole lot more sense :smiley:  :+1:

**Comment by sciunto** [comment]
yep :)

**Comment by wking** [comment]
Thanks for the fixup, and sorry for the slow reply :p.  I've squashed
your two commits into one and avoided wrapping the string literal with
8e277573 (also rebased around #64, although there were no conflicts
there).  If that looks good to you, feel free to force-push it to your
‘requirements’ branch to update this PR.  Or just let me know if I can
add your Signed-off-by [1] and I'll add it to my reroll and merge that
instead.

[1]: https://github.com/wking/rss2email/blob/master/CONTRIBUTING.md

**Comment by sciunto** [comment]
@wking Yes, sure, you can merge :)

**Comment by wking** [comment]
Merged the squashed commit in with a @sciunto Signed-off-by in 0489b58.

**Pull request closed without merging**